deps/Makefile

```diff
@@ -39,7 +39,7 @@ include $(SRCDIR)/tools/git-external.mk
 # prevent installing libs into usr/lib64 on opensuse
 unexport CONFIG_SITE
 
-DEP_LIBS :=
+DEP_LIBS := compiler-rt
```

Comment:
It is more complicated than that. As far as I can tell we will need to unconditionally build with compiler-rt (except for llvm-free builds).
`BUILD_COMPILER_RT==1` means that we will try to build a standalone compiler-rt iff LLVM hasn't build one.